app/build.gradle

```diff
@@ -68,6 +68,9 @@ android {
     androidExtensions {
         experimental = true
     }
+    testOptions {
```

Comment:
Did this change fix the Travis build issue, if yes then please share the reference?

Reply:
It is a recommended practice to disable animations in the device for testing. It is also mentioned [here](https://developer.android.com/training/testing/espresso/setup#set-up-environment) in the  documentation for espresso